aot/src/main/java/com/dylibso/chicory/experimental/aot/AotMachine.java

```diff
@@ -8,11 +8,23 @@
 /**
  * Machine implementation that compiles WASM function bodies to JVM byte code.
  * All compilation is done in a single compile phase during instantiation.
+ * <p>
+ * This class is deprecated and will be removed in a future version. Please use
+ * the {@link com.dylibso.chicory.compiler.MachineFactoryCompiler} instead.
  */
+@Deprecated(since = "1.4.0", forRemoval = true)
```

Comment:
I did the same, but, turns out that `forRemoval = true` is getting marked as an error by the IDEs, causing major confusion.